From 52e3e529077a634f6f184cfc992c1dff061f56fe Mon Sep 17 00:00:00 2001
From: ejaaskel <esa.jaaskela@suomi24.fi>
Date: Wed, 12 Nov 2025 13:43:48 +0200
Subject: [PATCH] policy/modules/system/logging: Allow auditd to create in
 /var/run

auditd needs permissions to create into the /var/run, so allow it.
---
 policy/modules/system/logging.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/system/logging.te b/policy/modules/system/logging.te
index 1feb3271b..38ad8372c 100644
--- a/policy/modules/system/logging.te
+++ b/policy/modules/system/logging.te
@@ -184,6 +184,7 @@ allow auditd_t auditd_log_t:dir setattr;
 manage_lnk_files_pattern(auditd_t, auditd_log_t, auditd_log_t)
 allow auditd_t var_log_t:lnk_file read_lnk_file_perms;
 allow auditd_t var_log_t:dir search_dir_perms;
+allow auditd_t var_run_t:dir create;
 
 manage_files_pattern(auditd_t, auditd_runtime_t, auditd_runtime_t)
 manage_sock_files_pattern(auditd_t, auditd_runtime_t, auditd_runtime_t)
-- 
2.34.1

